Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN addgroup --system --gid 1001 nodejs && \
 
 COPY --chown=nodejs:nodejs . .
 
+RUN rm -rf azure-mock/
+
```

Comment:
As discussed we will add each file and directory explicitly

Reply:
b00522c125bf14fa29c90ad180a7b8e5d80ec25a

I think this one is cleaner. To explicitly copy the files i need, i have to include all the config files for angular, express and typescript individually. Usually, that's what dockerignore is for. In this example, i cannot exclude the azure-mock/ directory via dockerignore because i need it for the migration dockerfile in a local scenario.

Using multi-stage-build and only copy the dist js code together with the package dependencies does:
- solve the issue with the azure-mock dir
- slims down the production image anyways to what's _really_ needed